resources/views/layouts/app.blade.php

```diff
@@ -19,13 +19,13 @@
     <body>
         <div class="ui secondary pointing menu bluemarketheader" id="bluemarketheader">
             <div class="right menu">
-                <a class="active item" href="/">
+                <a class="{{ Request::is('/') ? 'active item' : 'item' }}" href="{{ url('/') }}">
                     Home
                 </a>
-                <a class="item" href="/projects">
+                <a class="{{ Request::is('projects') ? 'active item' : 'item' }}" href="{{ url('projects') }}">
```

Comment:
Esto lo puedes acortar si sacas el item de los braces: `class="item {{ Request::is..."`